packages/core/test/ServerTest.ts

```diff
@@ -887,7 +887,7 @@ class WoTServerTest {
             expect(content.body).not.to.be.undefined;
 
             const body = await content.toBuffer();
-            body.should.be.eq('"test"');
+            body.toString().should.be.eq('"newValue"');
```

Comment:
I assume this is the correct assumption ?

Reply:
Yes correct and this was not failing without that option? this is quite bad from mocha side...

Reply:
Correct, it was not failing before.

Having said that, since the code is correct, just the test is not correct it is not as bad ...